core/server/worker/src/main/java/alluxio/worker/grpc/ShortCircuitBlockReadHandler.java

```diff
@@ -115,7 +115,7 @@ public void exceptionCaught(Throwable e) {
         mResponseObserver.onError(GrpcExceptionUtils.fromThrowable(e));
       }
     }, "OpenBlock", true, false, mResponseObserver, "Session=%d, Request=%s",
-        mSessionId, mRequest);
+        mSessionId, request);
   }
```

Comment:
```suggestion
  // Must use request object directly for this log as mRequest is only set in the Callable
  }
```

Reply:
Also I'm guessing the same workaround isn't needed for `onCompleted()` because `mRequest` gets set by `onNext()`?